CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+# 9.5.0
+
+### Added
+- Thanks to @muan, it's now possible to style `<summary>` elements as buttons and have them appear in the active/selected state when the enclosing [`<details>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details) is open. #346
```

Comment:
This feels inconsistent with our format of "added", "changed" etc. To thank people I'd rather have a "thanks to contributions from..." section for each release. For example `@JasonEtco` contributed to the last release and we didn't thank him 😢